### DISCUSSION TITLE
I can't remove code line numbers

### DISCUSSION THREAD

**Discussion by vanderfsouza** [quarto-dev/quarto-cli#5350] [Q&A]
Hello, for some reason I can't disable the line numbers in the **.yml** file: 
```
format:
  html:
    code-line-numbers: false
```

The other settings in the .yml file are working and I am not using this same attribute (code-line-numbers) in the chunks (there is no overlapping in this case).

**Comment by mcanouil** [comment]
I can't reproduce using either current stable version (1.3.340) and current dev version.
Could you provide a full reproducible example?
See <https://quarto.org/bug-reports.html>

**Reply by vanderfsouza** [reply]
It's working now. I think it's something related to **cache** or **freeze**, I deleted these files and got the expected result. Sorry for the inconvenience. We can delete this topic if you don't mind.
